lte/gateway/python/integ_tests/s1aptests/test_attach_detach.py

```diff
@@ -20,7 +21,8 @@
 class TestAttachDetach(unittest.TestCase):
 
     def setUp(self):
-        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self.mock_pcrf = {True if (os.environ.get("FEDERATED_MODE") == "True") else False}
```

Comment:
The curly braces must be removed for `self.mock_pcrf` to become boolean. Otherwise, it will be a set and the `if self.mock_pcrf` check always becomes true. Please do this for all tests.

Reply:
And I think this can be simplified to
```
self.mock_pcrf = os.environ.get("FEDERATED_MODE") == "True"
```